Radzen.Blazor/RadzenMonthView.razor

```diff
@@ -8,7 +8,9 @@
     {
         get
         {
-            return Scheduler.CurrentDate.ToString("MMMM yyyy");
+            var sb = new System.Text.StringBuilder(Scheduler.CurrentDate.ToString("MMMM yyyy", Scheduler.Culture));
+            sb[0] = char.ToUpper(sb[0], Scheduler.Culture);
```

Comment:
Why is this needed?

Reply:
Some cultures don't treat months as names, resulting in incorrect capitalization for this usecase, sv-SE is one of them.

Reply:
Shouldn't we leave it as it is per the Culture rules then? If a culture does not capitalize its month names we shouldn't do that as well.

Reply:
Sure, but in this case its also the start of a heading/sentence, which is why should be capitalized even though the months aren't treated as names.

Reply:
Let's remove that capitalization from the code. If needed it can be achieved with CSS.